### ISSUE TITLE
Deep_Learning_with_TensorFlow/1.0.0/Chapter10/2. 多GPU并行.py中某些代码还没有更新为1.0.0的版本

### ISSUE THREAD

**Issue by skx300** [caicloud/tensorflow-tutorial#34]
需要做以下更新：
```
tf.nn.sparse_softmax_cross_entropy_with_logits(y, y_) 　-> tf.nn.sparse_softmax_cross_entropy_with_logits(logits = y, labels = y_)

tf.histogram_summary() -> tf.summary.histogram()

tf.merge_all_summaries() -> tf.summary.merge_all()

tf.train.SummaryWriter() -> tf.summary.FileWriter()
```

**Comment by ScorpioCPH** [comment]
@skx300 welcome contributions, please feel free to fire a PR.

**Comment by weizhenzhao** [comment]
哥哥们 多GPU并行有人跑通了么？我跑了一半然后就一直包这个错呀
`input_producer/mul/y: (Const): /job:localhost/replica:0/task:0/cpu:0
input_producer/Assert/Assert/data_0: (Const): /job:localhost/replica:0/task:0/cpu:0
input_producer/Assert/Const: (Const): /job:localhost/replica:0/task:0/cpu:0
input_producer/Greater/y: (Const): /job:localhost/replica:0/task:0/cpu:0
input_producer/Size: (Const): /job:localhost/replica:0/task:0/cpu:0
input_producer/Const: (Const): /job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/mul/y: (Const)/job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/Assert/Assert/data_0: (Const)/job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/Assert/Const: (Const)/job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/Greater/y: (Const)/job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/Size: (Const)/job:localhost/replica:0/task:0/cpu:0
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\simple_placer.cc:827] input_producer/Const: (Const)/job:localhost/replica:0/task:0/cpu:0
step 10, loss = 33.57 ( 28495.4 examples/ sec; 0.004 sec/batch)
step 20, loss = 23.26 ( 28495.4 examples/ sec; 0.004 sec/batch)
step 30, loss = 34.08 ( 24932.0 examples/ sec; 0.004 sec/batch)
step 40, loss = 7.51 ( 28495.8 examples/ sec; 0.004 sec/batch)
step 50, loss = 12.70 ( 31005.2 examples/ sec; 0.003 sec/batch)
step 60, loss = 7.47 ( 28492.0 examples/ sec; 0.004 sec/batch)
step 70, loss = 17.77 ( 30687.6 examples/ sec; 0.003 sec/batch)
step 80, loss = 14.76 ( 28497.3 examples/ sec; 0.004 sec/batch)
step 90, loss = 14.28 ( 30686.5 examples/ sec; 0.003 sec/batch)`

**Comment by Hui-Zhi** [comment]
@lienhua34 PTAL
